multicluster/controllers/multicluster/commonarea/resourceimport_controller.go

```diff
@@ -146,6 +137,9 @@ func (r *ResourceImportReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		}
 		return r.handleResImpUpdateForClusterInfo(ctx, req, &resImp)
 	}
+	if isDeleted {
+		r.installedResImports.Delete(resImp)
+	}
```

Comment:
why this is needed? Doesn't it ignore LabelIdentity events?

Reply:
This is not related to LabelIdentity. This is to fix an existing issue that when resImp is added to installedResImports it is never deleted

Reply:
Noted. removed from this PR and will open a separate one to fix this issue